internal/grpc/services/gateway/gateway.go

```diff
@@ -124,12 +130,28 @@ func (c *config) init() {
 	}
 
 	// caching needs to be explicitly enabled
-	if c.CacheStore == "" {
-		c.CacheStore = "noop"
+	if c.StatCacheStore == "" {
+		c.StatCacheStore = "noop"
+	}
+
+	if c.StatCacheDatabase == "" {
+		c.StatCacheDatabase = "ocis"
```

Comment:
in reva the default should be `reva`
```suggestion
		c.StatCacheDatabase = "reva"
```